src/InvestmentManager.sol

```diff
@@ -11,31 +11,9 @@ import {IRoot} from "src/interfaces/IRoot.sol";
 import {IERC20, IERC20Metadata} from "src/interfaces/IERC20.sol";
 import {IPoolManager} from "src/interfaces/IPoolManager.sol";
 import {IInvestmentManager, InvestmentState} from "src/interfaces/IInvestmentManager.sol";
-
-interface GatewayLike {
```

Comment:
Here, and in a lot of other files, I removed the duplicate interfaces and just used the existing ones.